commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/QueryModelImplRules.java

```diff
@@ -189,7 +189,7 @@ public boolean accept(final ReferenceType referenceType) {
                     .filter(StringUtils::isNotEmpty)
                     .map(baseClass -> {
                         builder.setBaseClassName(baseClass);
-                        methodRules.addFirst(new IgnoreFields("customerId", "customerEmail", "totalPrice", "taxedPrice", "country", "customerGroup", "lineItems", "customLineItems", "shippingAddress", "billingAddress", "shippingInfo", "discountCodes", "paymentInfo", "anonymousId", "locale", "custom"));
+                        methodRules.addFirst(new IgnoreFields("customerId", "customerEmail", "totalPrice", "taxedPrice", "country", "customerGroup", "lineItems", "customLineItems", "shippingAddress", "billingAddress", "shippingInfo", "discountCodes", "paymentInfo", "anonymousId", "locale", "custom", "shippingRateInput"));
```

Comment:
Why do we have to modify this? I even didn't know before that this exists.

Reply:
if you don't add the attribute here there going to be non compilable `CartQueryModel` and `OrderQueryModel` i think the code generator succeeds only for some simple attributes and failes with others (the one with no default constructor ...)

Reply:
I see, but don't fully understand why we can't derive this from our `@IgnoreInQueryModel`. Let's discuss this further and see if we should create a separate issue for fixing our code generator.